cmd/fossa/init.go

```diff
@@ -0,0 +1,77 @@
+package main
+
+import (
+	"fmt"
+	"regexp"
+
+	logging "github.com/op/go-logging"
+	"github.com/urfave/cli"
+
+	"github.com/fossas/fossa-cli/builders"
+	config "github.com/fossas/fossa-cli/config"
+)
+
+var initLogger = logging.MustGetLogger("init")
+
+func initCmd(c *cli.Context) {
+	conf, err := config.New(c)
+	if err != nil {
+		initLogger.Fatalf("Could not load configuration: %s", err.Error())
+	}
+
+	if err := doInit(&conf, c.Bool("force"), c.Bool("include-all")); err != nil {
+		initLogger.Fatalf("Error initializing: %s", err.Error())
+	}
+
+	if !c.Bool("skip-write") {
+		if err := config.WriteConfigFile(&conf); err != nil {
+			initLogger.Fatalf("Error writing config: %s", err.Error())
+		}
+
+		initLogger.Warningf("Config for %d modules written to `%s`.", len(conf.Modules), conf.ConfigFilePath)
+	}
+
+	fmt.Println("`fossa` is initialized")
+}
+
+func doInit(conf *config.CliConfig, force bool, includeAll bool) error {
+	var err error
+	if len(conf.Modules) == 0 || force {
+		conf.Modules, err = findModules(".")
+		if err != nil {
+			initLogger.Warningf("A non-fatal error occured during module discovery: %s", err.Error())
+		}
+
+		if !includeAll {
+			// fitler suspicious modules
+			filteredModuleConfigs := []config.ModuleConfig{}
```

Comment:
You should use the `var slice []T` form unless the length is known, in which case use `make`.